### PR TITLE
Sphinx doc chapter 30

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

**Comment by Zimmi48** [comment]
What did you do with your key? Did you use a new one? Was it really wanted? If yes, you should inform GitHub about it but you shouldn't change your key like that without a good reason...

File: doc/sphinx/addendum/miscellaneous-extensions.rst [hunk 1]
```diff
@@ -1,63 +1,74 @@
...
+     reflexivity.
+
+     Qed.
+
+     End P.
+
+     Print p.
+
+     Check h.
+
```
Comment (Zimmi48) [inline doc/sphinx/addendum/miscellaneous-extensions.rst]: Why all the newlines?

**Review by Zimmi48 — COMMENTED** [review]
I wonder if it wouldn't make sense to rename this chapter since it's only about `Derive`.

**Comment by Zimmi48** [comment]
Sorry my comment about your key was wrong: it is just that your key is expired: you should set a new expiration date (e.g. in one or two years) and republish your key on the keyservers and on GitHub.

**Comment by maximedenes** [comment]
> Sorry my comment about your key was wrong: it is just that your key is expired: you should set a new expiration date (e.g. in one or two years) and republish your key on the keyservers

That was already done when you commented. Are you sure I need to "republish it on GitHub"?

**Comment by maximedenes** [comment]
Comments addressed, rebased on other Sphinx PRs.

**Comment by Zimmi48** [comment]
> Are you sure I need to "republish it on GitHub"?

Yep, because GitHub continues to show the commits as "unverified".

**Comment by maximedenes** [comment]
@Zimmi48 can you merge this one?

**Pull request merged**